dca/tl_page.php

```diff
@@ -16,6 +16,7 @@
 $GLOBALS['TL_DCA']['tl_page']['palettes']['__selector__'][] = 'cookiebar_enable';
 $GLOBALS['TL_DCA']['tl_page']['palettes']['__selector__'][] = 'cookiebar_analyticsCheckbox';
 $GLOBALS['TL_DCA']['tl_page']['palettes']['root'] .= ';{cookiebar_legend},cookiebar_enable';
+$GLOBALS['TL_DCA']['tl_page']['palettes']['rootfallback'] .= ';{cookiebar_legend},cookiebar_enable';
```

Comment:
Can you wrap this with the if statement please?

```php
if (isset($GLOBALS['TL_DCA']['tl_page']['palettes']['rootfallback'])) {
    $GLOBALS['TL_DCA']['tl_page']['palettes']['rootfallback'] .= ';{cookiebar_legend},cookiebar_enable';
}
```

Reply:
Yes of course. You're right.